### PR TITLE
ui changes

### PR DESCRIPTION
**Pull request by JuileeBhombe**
# Pull Request

## Description

Recipe Screen and Recipe Details Screen UI

### CODE REVIEW

File: JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        }
+        items(50) {
+            RecipeCard()
+        }
+    }
+}
+
+
+@Composable
+fun RecipesTab() {
```
Comment (anandwana001) [inline JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt]: fun can be private?

File: JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+        Image(
+            modifier = Modifier.fillMaxSize(),
+            painter = painterResource(id = R.drawable.ic_launcher_background),
+            contentDescription = "Background Image", contentScale = ContentScale.FillBounds
+        )
+
+        Text(
+            text = "Recipe Name", modifier = Modifier.padding(15.dp), fontWeight = FontWeight(900),
+            fontSize = 20.sp,
+            letterSpacing = (1).sp,
```
Comment (anandwana001) [inline JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt]: any specific reason for brackets? `(1).sp`

File: JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipeDetails.kt [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.juileebhombe.reciperoulette.R
+import com.juileebhombe.reciperoulette.ui.theme.RecipeRouletteTheme
+
+@Composable
+fun RecipeDetails() {
+    Column(Modifier.verticalScroll(rememberScrollState())) {
```
Comment (anandwana001) [inline JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipeDetails.kt]: this vs LazyColumn?
any rational for this?

**Review by anandwana001 — CHANGES_REQUESTED** [review]
Can we add screenshots?

**Comment by JuileeBhombe** [comment]
> Can we add screenshots?

added in readme file

File: JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt (hunk 1 above)
Comment (JuileeBhombe) [reply JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt]: Resolved

File: JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt (hunk 2 above)
Comment (JuileeBhombe) [reply JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipesScreen.kt]: Resolved

File: JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipeDetails.kt (hunk 3 above)
Comment (JuileeBhombe) [reply JuileeBhombe/RecipeRoulette/app/src/main/java/com/juileebhombe/reciperoulette/ui/screens/RecipeDetails.kt]: Resolved

**Review by anandwana001 — APPROVED** [review]

**Pull request merged**